### DISCUSSION TITLE
Connecting with Homenetwork failed

### DISCUSSION THREAD

**Discussion by Fenar91** [doudar/SmartSpin2k#411] [Q&A]
Hello Guys,
i just sold one unit to a customer but he just cant connect his Smartspin with the home network. We dont really understand the problem... He has the right networkfrequenzy of 2,4 ghz and didnt misstyped according to him neither the ssid nor the password.. The reflashing didnt helped either. The following link shows his networkinformations           

[removed password info]
SSID `FRITZ!BOX Fon WLAN 2.4`

Does anybody has any idea and can help us?

Greetings from Germany.

**Comment by doudar** [comment]
The firmware doesn't gracefully handle spaces in the SSID right now. It's on my list to fix soon.

**Reply by Fenar91** [reply]
He changed according to him the ssid to one without handle space and it didnt work either... Im really confused.

**Reply by doudar** [reply]
Can he connect to the SmartSpin2k (default) network?

If not it might be a bad ESP32 devkit. There are a *ton* of bad devkits out there right now with this issue. @eMadman is experiencing nearly 30% failure rate on devkits from China. This is one thing that's spurred us to create our own board design which should be released very shortly. 

Otherwise, the only WiFi issue I know of are the SSID space issue, SS2k won't attempt to reconnect to WiFi if the connection is interrupted, and the ESP32 we are using isn't compatible with 5ghz networks (2.4 only.)

**Reply by Fenar91** [reply]
Hi,
i think he did connect to the default network as u can see on the picture. So he did see the Smartspin2k network signal.

**Reply by doudar** [reply]
Good point! 🤣 

Maybe @eMadman can fill in some additional information. I'm at a loss for answers based on your information. It should work.

**Reply by Fenar91** [reply]
XD

**Reply by doudar** [reply]
Btw, are you looking at building many of these? If so, we could get you added to our links. The world could use a source in Europe!

**Reply by Fenar91** [reply]
Hi douder,
i sold 2 in 2 months but i dont think it is worth the time, because i charged only 100 € per unit with my own switch and bracket design (see link). And i dont think there is a lot of potential costumer here. What is ur opinion? I can let my physicstudents build it like a sclave. But they would learn something too :D
https://ibb.co/CWZ7Ffs
https://ibb.co/8M9yWsd
https://ibb.co/5W2P3t6

**Reply by Fenar91** [reply]
Should i build more? How big is the demand in europe?

**Reply by doudar** [reply]
I'm not certain - I'm guessing you could build as many as you desire. It's hard to know without a sales link. Make some sort of shop link and I'll post it on our kits page if you're interested.

**Reply by Fenar91** [reply]
U guys are really great, thank u so much for the offer. I will consider it!

**Reply by Fenar91** [reply]
Hi Doudar,
how can i contact u so i can send u the sales link only for a part of europe?

**Reply by doudar** [reply]
smartspin2k@gmail.com
